internal/core/domain/stats_model.go

```diff
@@ -0,0 +1,31 @@
+package domain
+
+// Withdrawal is used to follow funds withdrawal statistics
+type Withdrawal struct {
+	ID              uint `badgerhold:"key"`
```

Comment:
domain MUST not have any reference to any kind of concrete db whether it is badger, postgres or whatever. Please remove. Also, a withdrawal is already uniquely identified by the tx id included in blockchain, therefore there's no need to have another one.

Reply:
Done